mindsdb/integrations/handlers/twitter_handler/twitter_handler.py

```diff
@@ -312,6 +312,7 @@ def call_twitter_api(self, method_name:str = None, params:dict = None):
                 else:
                     params['max_results'] = left
 
+            log.logger.debug(f'>>>twitter in: {method_name}({params})')
```

Comment:
Same as the above, maybe there is a cleaner alternative to `>>>`, but this is not critical